### ISSUE TITLE
Frontend design fixes

### ISSUE THREAD

**Issue by NicoSerranoP** [zkparty/trusted-setup-frontend#84]
- [x] Entropy Input: prevent input size change when entering entropy string
- [x] Landing: "learn more about" is not working

**Comment by tvanepps** [comment]
1. the **`Submit`** button activates before the user has added their text entropy.

![image](https://user-images.githubusercontent.com/27454964/203570769-aabf385e-7101-403f-8c32-441cf3515b67.png)

2. to illustrate the size change/ movement in the entropy box (i can re-export components if you need new ones):

![image](https://user-images.githubusercontent.com/27454964/203565425-4a5e1721-99b0-441d-b0d5-e36c4c8081cd.png)

3. the snake graphic has misalignments and color bleeding out. the biggest one is the black line where the head meets the top of the body

![image](https://user-images.githubusercontent.com/27454964/203567349-ec8df0ba-5c11-492d-aab2-521ef768963c.png)

4. on the **Bind your Contribution** step - there is a misalignment of components, shouldn't be any white showing between the rings. similarly for between the tan circle 

![image](https://user-images.githubusercontent.com/27454964/203567080-99f2d180-4205-4913-969d-be8a23f57938.png)

5. can we remove the spinning animation on the **Waiting to be Submitted** for now? i think it needs to either be slowed down or replaced with something more gentle, like a pulse. it's also strange that the black and white version is still visible underneath. The title and body text are not vertically centered in the white space, possibly bc we add the error message in? We should consider removing this error message, as the user already knows they need to wait by the regular copy. Finally, if an error message is needed, the two text boxes are not aligned on the left side

![image](https://user-images.githubusercontent.com/27454964/203569498-b50dc861-324c-46e7-84b9-910212cee41a.png)

last thing, it's strange that the full illustration gets cut off when adding back in the supplemental information about danksharding below. should we consider not clipping this?

![image](https://user-images.githubusercontent.com/27454964/203571248-e970e35d-1be1-47cb-a8db-cd6668d7cd1c.png)

6. I have two questions here - doesn't SIWE mean we don't need to repeat the login process each time as long as the session is still active? Second, if we are going to persist information, there should be a way for them to **log out** an address out in case they want to use a different one

![image](https://user-images.githubusercontent.com/27454964/203570464-c95403e3-1926-4d93-a1ec-c80a6c7b2017.png)

**Comment by NicoSerranoP** [comment]
@tvanepps I got most of the observations finished but we should consider the following:

3. I cannot fix this because it is part of the dynamic `SnakeProgress` component that gets colored with the mouse movement. We need to reexport this graphic and check out with SVGator (the animation library) to see how we fix the misalignments in the snake head & body. Maybe @tkmct could you give us a hint?

5.1. Wallet Choosing: I was not able to make the background blurred because it is an external component that doesn't let us set many options. I was able to make the background darker so the back image is not that visible. What do you think?
![image](https://user-images.githubusercontent.com/38594836/204722222-2c3d9c67-472f-4d61-8115-d16118b65d86.png)

6. I increase the animation time from 12s to 40s in the waiting lobby. The rounding animation is slower now. The pulse animation is applied when the contribution is being computed. I think a more explicit animation (rounding) is better to notify the user he might have to wait a bit in the lobby.

7. The SIWE is done between the user, the external OIDC (SIWE) and the sequencer. After the exchange, the sequencer gives the user a session id to use it as an authentication method for the other routes (lobby and contribution phases). The double signing re-asks the user to sign a message because it is done between the user, the wallet provider and the frontend. After the signature, the frontend sends the signed message along with the contribution to the sequencer. 

8. We could keep the main navbar (logo in the left and language switcher on the right) but it would cut out a piece of the pizza because its height. I feel we can leave the back arrow redirecting to home but what do you think?
![image](https://user-images.githubusercontent.com/38594836/204723713-fd12b2f2-4fc3-4d10-9b06-c315aebae42c.png)

**Comment by tvanepps** [comment]
@NicoSerranoP - your replies make sense, thanks for looking into my suggestions! 

3. we can address this in a separate PR when we have the proper asset
5.1 understood, darker is fine
6. 👍 
7. 👍 
8. still thinking about this one. leave as is for now 🙏
